examples/angular/package.json

```diff
@@ -4,31 +4,31 @@
     "description": "Demo of building Angular apps with Bazel",
     "license": "Apache-2.0",
     "engines": {
-        "node": ">=10.9.0 <13.0.0",
+        "node": ">=10.9.0 <=13.8.0",
```

Comment:
@gregmagolan I hope this was ok? I had 13 locally and was just testing this out 🤷‍♂

Reply:
Noticed that and LGTM. Its still the supported range for rules_nodejs & for this example.